sweagent/agent/agents.py

```diff
@@ -571,7 +601,16 @@ def forward(self, observation: str, available_actions: list[str], state: str) ->
             action: action that the model proposes
             output: raw model output (not output of the action)
         """
-        thought, action, output = self.forward_with_error_check(observation, state)
+        if self.initial_model_response is not None:
+            thought, action, content = self.initial_model_response["thought"], self.initial_model_response["action"], self.initial_model_response["content"]
+            if isinstance(content, str):
+                output = content
+            else:
+                output = AnthropicModelResult(blocks=content)
```

Comment:
FYI: I have been trying to keep all explicit mentions of `AnthropicModelResult` to the utility functions in `models.py` (and the Anthropic-specific parser). I'd recommend moving it there, so we don't lose track of all the relevant locations.